mxcube3/routes/Queue.py

```diff
@@ -234,7 +230,7 @@ def execute_entry_with_id(node_id):
         (any further error might still happen)
         :statuscode: 409: queue entry could not be executed
     """
-    last_queue_node = mxcube.queue.last_queue_node
+    # last_queue_node = mxcube.queue.last_queue_node
```

Comment:
Hey

I noticed this when I skimmed trough the file, its maybe something that remains after the pep8 work.  It has of course no impact but thought it was better to let you know.